myuw/templates/handlebars/textbooks.html

```diff
@@ -3,198 +3,202 @@
-                        {{/if}}
-
-                        <dl class="dl-books">
-                            <dt>Author</dt>
-                            {{#each authors}}
-                            <dd>{{name}}</dd>
-                            {{/each}}
-
-                            <dt>New </dt>
-                            <dd>
-                                {{#if price}}
-                            ${{formatPrice price}}
-                                {{else}}
-                                *
-                                {{/if}}
-                            </dd>
-
-                            <dt>Used</dt>
-                            <dd>
-                                {{#if used_price}}
-                                ${{formatPrice used_price}}
-                                {{else}}
-                                *
-                                {{/if}}
-                            </dd>
-
-                            {{#if notes}}
-                            <dt>Notes</dt>
-                            {{#if is_required}}
-                                    <dd>Required</dd>
-                            {{else}}
-                                <dd>{{notes}}</dd>
-                                {{/if}}
-                            {{/if}}
-                        {{#if isbn}}
-                                <dt>ISBN</dt>
-                            <dd>{{isbn}}</dd>
-                        {{/if}}
-                        </dl>
-                    </div>
-                {{/eachWithIndex}}
-
-            {{else}}
-                <div id="books{{sln}}" class="{{#if @root.collapse_sections}}collapse{{/if}}" style="margin-bottom:25px;">
-                  No textbook requirement has been received for this course.
-                  Please check with your instructor.
-                </div>
-
-            {{/if}}
-        </div> <!-- end booklist -->
-
-
-        {{#if @root.collapse_sections}}
-        {{else}}
-            <hr />
-        {{/if}}
-    {{/each}}
-
-
-    {{#if verba_link }}
-    <div style="text-align:center; margin-bottom:10px;"><a target=_blank href="{{verba_link}}"><button class="btn btn-default" type="button">Compare Prices and Buy Books</button></a></div>
-    <!-- <p class="text-muted">Read the University Book Store's <a href="http://uw-seattle.verbacompare.com/" target="_blank">price comparison policies</a></p> -->
-    {{/if}}
-
-    <div style="margin-top:20px;">
-	<p class="text-muted">Information on course textbooks is collected by and provided courtesy of <a href="http://www.bookstore.washington.edu">University Book Store</a> and is subject to change regularly and without notice.</p>
+    <div class="full-page-error">
+        <p class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>Sorry, there is a problem getting data from University Bookstore. We are working with them to fix the issue. </p>
```

Comment:
The card level error message style-- "An error has occurred and MyUW cannot display textbook information right now. Please try again later."